### ISSUE TITLE
Reset switches hard to source

### ISSUE THREAD

**Issue by ThomasAlban** [raeedcho/chocofi-temper#19]
Hi,
It’s practically impossible for me (uk) to source the reset button which is used on this keyboard without paying extortionate shipping. The one on the original chocofi (pts636) is much more easily available. Is there any chance of a version of this keyboard being made that supports that type of reset switch?
Thanks

**Comment by raeedcho** [comment]
Ah, that is frustrating--I didn't realize that when designing the board. Unfortunately, I don't currently have plans to make a second version of the temper, but I wouldn't rule it out, if there are enough desired changes to make it worthwhile. Sorry! I'll leave this issue open for now as a feature request and let others comment.

**Comment by ThomasAlban** [comment]
Yeah that would be great.
I'm really keen to build the temper as it seems like pretty much my ideal keyboard (I currently use a corne but I don't need the outer row of keys and I find the little finger stagger not low enough)
When I build it I'd like to add 1000mah batteries below the keyboard, so if I solder the battery connectors to the pads on the bottom of the pcb would it still work? (So I don't have to have wires protruding out of the keyboard)
If not, perhaps another modification could be a little indent above the battery pads so the wires can wrap around without protruding.

**Comment by raeedcho** [comment]
Yes, you should be able to just solder the batteries onto the backside of the board. I would double check with a multimeter to make sure, but the PCB is designed so that the pads on front and back are directly connected.

**Comment by AdrianBonpin** [comment]
Quick question, regarding the reset switch as I'm not proficient with reading pcbs (which I need to really learn). Will standard 4 pin reset switches work? Specifically these ones that I can source.
<img width="208" alt="image" src="https://github.com/user-attachments/assets/f9f6772a-fe65-4986-ab3d-d1093e9d2514" />

I apologize if my question has a simple answer or may be mistaken and I thank you in advance. (And an advanced Happy New Year)

**Comment by raeedcho** [comment]
I'm sorry to say that I'm not sure this will fit in the same reset switch spot--it kind of looks like the contacts are not quite positioned correctly to sit on the PCB pads, but I can't be certain.

**Comment by AdrianBonpin** [comment]
Footprint aside as I can simply play around with the placement, does it function the same? but just with 4 pads?

I was trying to trace where the 4 pads lead to but I really couldn't figure it out. Thanks again

**Comment by raeedcho** [comment]
As far as I know, both are just momentary switches that shorts the pairs of pads when pressed. As for which pads get connected on press, you'd have to look at the data sheet of your reset switch and match it to the PCB.

Basically though, when the reset button is pressed, the reset pin from the microcontroller should be connected to ground. So on the PCB, one pair of pads is routed directly to the ground plane of the PCB and one pair of pads is routed to the reset pin for the microcontroller.

Let me know if this makes sense--sorry I can't be of more help!

**Comment by szabolcsfuzi** [comment]
Hey, 
I just finished building (thanks @raeedcho) using a switch from [Aliexpress](https://vi.aliexpress.com/item/1005002845025014.html) (1005002845025014 item id).
